<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Ekran_start">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_log">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="1">
         <widget class="QWidget" name="Program" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="IleSlowek">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <property name="text">
                <string>Wybierz ilość słówek do powtórki</string>
               </property>
               <property name="textFormat">
                <enum>Qt::RichText</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSplitter" name="splitter">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="frameShape">
                <enum>QFrame::HLine</enum>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QSlider" name="horizontalSlider_Ilosc_Slow">
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>20</number>
                </property>
                <property name="sliderPosition">
                 <number>20</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
               <widget class="QSpinBox" name="spin_Ilosc_Slow">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
                <property name="displayIntegerBase">
                 <number>10</number>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="JakiPoziom">
               <property name="text">
                <string>Wybierz poziom trudności</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSplitter" name="splitter_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QSlider" name="horizontalSlider_Poziom">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>3</number>
                </property>
                <property name="value">
                 <number>2</number>
                </property>
                <property name="sliderPosition">
                 <number>2</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
               <widget class="QSpinBox" name="spin_Ilosc_Poziom">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>3</number>
                </property>
                <property name="value">
                 <number>2</number>
                </property>
                <property name="displayIntegerBase">
                 <number>10</number>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Rozpocznij naukę</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_data">
       <widget class="QPushButton" name="Dobierz_literke">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>181</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Dobierz literkę</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>721</width>
          <height>109</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="Pl_Slowko">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>MV Boli</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Hello</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="NoweSlowko">
             <property name="text">
              <string>Nowe słówko</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="sl_ang">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>400</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>MV Boli</family>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Ang_sl</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="Odpowiedz">
             <property name="maximumSize">
              <size>
               <width>1000</width>
               <height>35</height>
              </size>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="tabChangesFocus">
              <bool>true</bool>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ZatwierdzSlowo">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Zatwierdź słowo</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="Podsumuj">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>180</y>
          <width>171</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>Podsumowanie</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>282</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QSplitter" name="splitter_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Punktów do zdobycia</string>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::NoTextInteraction</set>
            </property>
           </widget>
           <widget class="QLabel" name="ilepunktow">
            <property name="inputMethodHints">
             <set>Qt::ImhDigitsOnly</set>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSplitter" name="splitter_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Suma punktów</string>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::NoTextInteraction</set>
            </property>
           </widget>
           <widget class="QLabel" name="Suma">
            <property name="inputMethodHints">
             <set>Qt::ImhDigitsOnly</set>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QSplitter" name="splitter_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>310</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Pozostałych słówek</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_2"/>
       </widget>
       <widget class="QSplitter" name="splitter_7">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>310</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Słówka do powtórki</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_3"/>
       </widget>
      </widget>
      <widget class="QWidget" name="page_end">
       <widget class="QSplitter" name="splitter_3">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>60</y>
          <width>171</width>
          <height>71</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Zdobyte punkty</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>120</red>
               <green>120</green>
               <blue>120</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font">
          <font>
           <family>Snap ITC</family>
           <pointsize>16</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="digitCount">
          <number>3</number>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Filled</enum>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>150</y>
          <width>201</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Jeszcze raz</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Zakończ</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionWczytaj_s_ownik"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionO_Qt_2"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionWczytaj_s_ownik">
   <property name="text">
    <string>Wczytaj słownik</string>
   </property>
  </action>
  <action name="actionO_Qt_2">
   <property name="text">
    <string>O Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_Ilosc_Slow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_Ilosc_Slow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_Ilosc_Poziom</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_Poziom</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_Ilosc_Slow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_Ilosc_Slow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Poziom</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_Ilosc_Poziom</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
